app/domain/output_document/html_renderer.rb

```diff
@@ -0,0 +1,51 @@
+class OutputDocument
+  class HTMLRenderer < SimpleDelegator
+    attr_accessor :output_document
+
+    def initialize(output_document)
+      @output_document = output_document
+      __setobj__(output_document)
+    end
+
+    def pages_to_render
```

Comment:
Similar to @andrewgarner's question above - should this be exposed publicly like this?